package.json

```diff
@@ -38,7 +38,8 @@
   "resolutions": {
     "**/browserslist": "^4.14.0",
     "**/fastboot": "^3.1.0",
-    "**/qunit": "^2.14.1"
+    "**/qunit": "^2.14.1",
+    "socket.io": "4.2.0"
```

Comment:
i'll revert this change once socket.io releases a fix

Reply:
ref: https://github.com/socketio/socket.io/issues/4121#issuecomment-943832512
ref: https://github.com/socketio/socket.io-client/pull/1503